### PR TITLE
Volumes from snaps doc update

### PR DESCRIPTION
**Pull request by andrijapanicsb**

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Comment by andrijapanicsb** [comment]
The new section here: https://acs-www.shapeblue.com/docs/pr59/adminguide/storage.html#creating-a-new-volume

@nvazquez @anuragaw @shwstppr - kindly review and LGTM.

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -230,6 +230,15 @@ are not allocated on the physical storage device until you attach the
 volume. This optimization allows the CloudStack to provision the volume
 nearest to the guest that will use it when the first attachment is made.
 
+When creating a new volume from an existing ROOT volume snapshot,
+it is required to explicitely define a Disk offering (UI will offer only Disk
```
Comment (PaulAngus) [inline source/adminguide/storage.rst]:
```suggestion
it is required to explicitly define a Disk offering (UI will offer only Disk
```

File: source/adminguide/storage.rst [hunk 2]
```diff
@@ -230,6 +230,15 @@ are not allocated on the physical storage device until you attach the
...
+offerings whose disk size is equal or bigger than the size of the snapshot).
+
+|volume-from-snap.PNG|
+
+When creating a new volume from an existing DATA volume snapshot, Disk offering
```
Comment (PaulAngus) [inline source/adminguide/storage.rst]:
```suggestion
When creating a new volume from an existing DATA volume snapshot, the disk offering
```

**Comment by andrijapanicsb** [comment]
requesting docbuild

**Pull request merged**